conftest.py

```diff
@@ -64,6 +71,43 @@ def plot_search_test_data(
091493 60.225389886889, 24.9303589737418 60.2253886208538, 24.9303635078267 60.2253872424947, 24.9303679004809 60.2253857551325, 24.9303721411221 60.2253841623503, 24.9303762195342 60.2253824679853, 24.930380125892 60.2253806761194, 24.9303834056943 60.2253790262418, 24.9309438731481 60.2250846622875, 24.9311492769493 60.2249247606672, 24.9313449058512 60.2250058341877, 24.9314799048349 60.225061780105, 24.9334995943141 60.2260142690243, 24.9337632460662 60.2261386010807, 24.933818351935 60.226164588021, 24.9338157864245 60.2261267869928, 24.9338927483051 60.226109481358, 24.9341019812378 60.2260624333412, 24.9346116804935 60.2259478205358, 24.9380966718863 60.2273153439199, 24.9383702130947 60.2277146432983, 24.9382265882778 60.2282711207616, 24.9381392978166 60.2286093094113, 24.9384032947399 60.2289240805279, 24.9383378519653 60.2290407944473, 24.9382113952431 60.2293030450244, 24.9384889631233 60.2294313681294, 24.938799962067 60.229159950719, 24.9394874223171 60.2293198478512, 24.9396900119737 60.2294841550461, 24.9390948607793 60.2296588803352, 24.9387914632488 60.2297479558305, 24.9383874013375 60.2298665739024, 24.9372095679121 60.2302092575693, 24.9371249463398 60.2302350611558, 24.9370683442711 60.230189152294, 24.9368848107175 60.2300402940762, 24.9356635452658 60.2298767088686, 24.9355825594403 60.2298658605598, 24.934379059859 60.2297046515136, 24.9339948778353 60.2296531874643, 24.9339919064033 60.2295944476404, 24.9339865207418 60.2294879657768, 24.9331467439544 60.2294792162818, 24.9330671109783 60.229477414347, 24.9329877517143 60.2294736711778, 24.9329088573596 60.2294679957925, 24.9328306179914 60.2294604018645, 24.9327532221081 60.2294509076897, 24.9326768561757 60.2294395361419, 24.9326017041784 60.2294263146182, 24.9325279471747 60.2294112749725, 24.9324557628621 60.2293944534391, 24.9323853251477 60.2293758905454, 24.9323208952348 60.2293569051354, 24.9315339726876 60.2291126301885, 24.9325999661542 60.2282630924284, 24.9312079020712 60.2278309845408, 24.9313368581088 60.2277282134113, 24.9299591210016 60.227300521052, 24.9297601184794 60.227238742562, 24.9296930117233 60.2272922199406, 24.9292256570319 60.2276646519495, 24.9284831194396 60.2274341327472, 24.9279177290487 60.2278087376594, 24.9277054735462 60.2278909379512, 24.9275697385973 60.2279969132392, 24.9276458337081 60.228021025773, 24.9274544054548 60.2281704831243, 24.9279408678184 60.228806389578, 24.9282827349743 60.2289220248337, 24.928238466007 60.2289544010617, 24.9277753139783 60.2292931169049), (24.9322140697331 60.2264591835492, 24.932023458066 60.2264031460676, 24.9318386046904 60.2263425237164, 24.9317181912374 60.2262994119744, 24.9304098788808 60.2273291023801, 24.9304741680556 60.2273594041884, 24.9305413692221 60.2273881004526, 24.9306113204842 60.2274151220367, 24.930683853319 60.2274404038389, 24.9307274500755 60.2274543743202, 24.9317178872309 60.2277618261751, 24.9327573873027 60.2266100800325, 24.9322140697331 60.2264591835492)))"  # noqa: E501
```

Comment:
Nitpicking again here, but coordinate pairs could be in a string array and using join & concatenate/string formatting would make this a bit tidier.

Reply:
Maybe even multipolygonfactory, so we can use these with other GEOS-related tests also?

Reply:
I have no idea what to do with this. I thought this is horrible way to do this but I have no idea how I should do this more clever way

Reply:
I would just put the coordinate pairs into an array first.

Then build that million mile long string by joining the coordinates from mentioned array? I guess it is more readable that way.